src/main/java/ru/practicum/shareit/item/dto/CommentOutDTO.java

```diff
@@ -0,0 +1,27 @@
+package ru.practicum.shareit.item.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class CommentOutDTO {
```

Comment:
Старался следовать общей концепции при использовании паттерна DTO. 
https://habr.com/ru/articles/513072/

> _Мы создаем по одному файлу для каждого контроллера, который содержит базовый enum без значений, в нашем случае это ProductDTO. Внутри него, мы разделяем DTO на те, что относятся к запросам (Request) и на те, что относятся к ответу (Response). На каждый endpoint мы создаем по Request DTO и столько Response DTO сколько нам необходимо. В нашем случае у нас два Response DTO, где Public хранит данные для любого пользователя и Private который дополнительно содержит оптовую цену продукта._

Мне показалось хорошей практикой стандартизировать разработку при использовании DTO. Но поскольку нашёл информацию только при работе с ФП 14 спринта и сжатыми сроками в моём случае, вероятно не много "криво" применил такой подход. Тем не менее, при доработке, считаю такой подход наиболее верным, если его чутка отполировать, то должно и выглядеть более понятно с точки зрения читаемости и использование с точки зрения функциональности должно быть более удобным

Reply:
Молодец, что читаешь хабр и изучаешь вопрос=) Но пойми главное программирование - это баланс. Подход хороший, когда у тебя в одном DTO 10 полей, в другом 20 и все еще разные. Но когда поля одинаковые и отличие всего в одно поле, которое можно проигнорить, то такой подход избыточный. Опять же, думать о развитии и дальнейшем пути оптимизации - это хорошо. Но это не должно быть в ущерб текущему приложению. Типа сейчас у нас не очень понятно, но вот когда станем гуглом, то наш подход принесёт плоды (спойлер - не принесёт, так как гуглом не станете=)). Так и в случае с петпроектом такой подход будет избыточным. https://www.baeldung.com/java-dto-pattern
Вот как раз про баланс:
The first mistake is to create different DTOs for every occasion. That will increase the number of classes and mappers we need to maintain. Try to keep them concise and evaluate the trade-offs of adding one or reusing an existing one.

We also want to avoid trying to use a single class for many scenarios. This practice may lead to big contracts where many attributes are frequently not used.

Ну и на последок, про применение технологий/паттернов и тд и тд:
https://taskinoor.wordpress.com/2011/09/21/the-abuse-of-design-patterns-in-writing-a-hello-world-program/